modules/tide_oauth/composer.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
We are still on Drush 8, does this present an issue here?

Reply:
it's not an issue. `drush.services.yml` only takes effect from Drush 9. That snippet ensures that we support both 8 and 9.

Reply:
👍